mkosi/__init__.py

```diff
@@ -1416,6 +1416,13 @@ def configure_dracut(args: CommandLineArguments, root: str) -> None:
     ):
         with open(os.path.join(dracut_dir, "30-mkosi-uefi-stub.conf"), "w") as f:
             f.write("uefi_stub=/usr/lib/systemd/boot/efi/linuxx64.efi.stub\n")
+
+    # efivarfs must be present in order to GPT root discovery work
+    if args.esp_partno is not None and args.distribution in (
+        Distribution.debian,
```

Comment:
I'm always wary of guards for Debian, but not Ubuntu. Is it correct, that it's not needed for Ubuntu?

Reply:
I'm 99% sure the original bug I found with booting only occurred with Debian and not Ubuntu.